luatest/server.lua

```diff
@@ -141,16 +141,6 @@ function Server:initialize()
 
     if self.workdir == nil then
         self.workdir = fio.pathjoin(self.vardir, self.id)
-        fio.rmtree(self.workdir)
```

Comment:
I would like to see a motivation of removing of the `rmtree` call.

Reply:
This call will be removed in the scope of a separate PR.